.github/workflows/cre-local-env-tests.yaml

```diff
@@ -100,7 +123,40 @@ jobs:
           E2E_TEST_CHAINLINK_VERSION: ${{ github.event_name == 'pull_request' && format('nightly-{0}-plugins', steps.set-date.outputs.date) || inputs.chainlink_image_tag }}
         run: |
           cd core/scripts/cre/environment
-          go run main.go env start
+          { time go run main.go env start; } 2> >(tee /tmp/time_output >&2)
+
+          # Convert time to total seconds
+          time_str=$(cat /tmp/time_output | grep real | awk '{print $2}')
+          minutes=$(echo $time_str | awk -F'm' '{print $1}')
+          seconds=$(echo $time_str | awk -F'm' '{print $2}' | sed 's/s//')
+          total_seconds=$(echo "$minutes * 60 + $seconds" | bc)
+          echo "execution_time=$total_seconds" >> $GITHUB_OUTPUT
+
+      - name: Track local env startup success rate
+        # track only scheduled runs to exclude failures in PRs or manual runs
+        if: always() && github.event_name == 'workflow_call'
+        shell: bash
+        run: |
+            success=false
+            if [ "${{ steps.start-cli.outcome }}" == "success" ]; then success=true; fi
```

Comment:
Same note here as earlier with double brackets and only using env vars in bash. Also is .outcome the right output? SHouldn't outputs always be under `.outputs`?